src/main/interfaces/UserResultData.ts

```diff
@@ -0,0 +1,22 @@
+import {PageData} from './PageData';
+
+export interface UserResultData extends PageData {
+  search: string;
+  result?: User;
+}
+
+export interface User {
```

Comment:
Maybe it'd be a good idea to separate User into it's own interface file as it's likely we'll probably use this data object elsewhere

Reply:
The User interface has now been separated into its own file